src/aws/cloudfront.tf

```diff
@@ -0,0 +1,97 @@
+locals {
+  s3_static_origin_id = "fun-mooc-static-origin"
+  s3_media_origin_id = "fun-mooc-media-origin"
+}
+
+# Create an origin access identity that will allow CloudFront to access S3
+# See bucket policies in s3.tf or documentation for more details:
+# https://www.terraform.io/docs/providers/aws/r/cloudfront_origin_access_identity.html
+resource "aws_cloudfront_origin_access_identity" "fun_mooc_oai" {
+  comment = "fun-mooc origin for the ${terraform.workspace} environment"
+}
+
+resource "aws_cloudfront_distribution" "fun_mooc_cloudfront_distribution" {
+  # Origin for the static S3 bucket
+  origin {
+    domain_name = "${aws_s3_bucket.fun_mooc_static.bucket_domain_name}"
+    origin_id   = "${local.s3_static_origin_id}"
+
+    s3_origin_config {
+      origin_access_identity = "${aws_cloudfront_origin_access_identity.fun_mooc_oai.cloudfront_access_identity_path}"
+    }
+  }
+
+  # Origin for the media S3 bucket
+  origin {
+    domain_name = "${aws_s3_bucket.fun_mooc_media.bucket_domain_name}"
+    origin_id   = "${local.s3_media_origin_id}"
+
+    s3_origin_config {
+      origin_access_identity = "${aws_cloudfront_origin_access_identity.fun_mooc_oai.cloudfront_access_identity_path}"
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+
+  # Allow public access by default, served by static bucket
+  default_cache_behavior {
```

Comment:
In the django's `storage.py` file, for static configuration you defined:
- `location = settings.STATIC_URL`

So I think you should you the `path_pattern` setting and set it to `/static/*`, WDYT ?

Reply:
Or maybe create an other `ordered_cache_behavior` dedicated to `static` files ?

Reply:
the static is covered by default behavior. Otherwise what would the default behavior be?

Reply:
is it mandatory to have one ? Does it make sense to serve something else than `media` and `static` ?

Reply:
Yep:
> default_cache_behavior (Required) - The default cache behavior for this distribution (maximum one).

Reply:
ok then keep the conf as it.

Reply:
What will happen is that you will hit the static bucket for anything else and get a 404... If we don't add anything else to the static bucket it should be ok?